### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**
[DEMO LINK](https://Shevtsooov.github.io/react_todo-app-add-and-delete/)

### CODE REVIEW

File: src/types/ErrorMessage.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessage {
+  noError = '',
+  loadError = 'Unable to load a todo',
+  addError = 'Unable to add a todo',
+  deleteError = 'Unable to delete a todo',
+  updateError = 'Unable to update a todo',
+  titleError = 'Title can\'t be empty',
+}
```
Comment (growingananas) [inline src/types/ErrorMessage.ts:8]: great!

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,166 @@
...
+import { ErrorMessage } from './types/ErrorMessage';
+import { TodoStatus } from './types/TodoStatus';
 
-const USER_ID = 0;
+const USER_ID = 10890;
 
 export const App: React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<TodoStatus>(TodoStatus.all);
+  const [isError, setIsError] = useState<ErrorMessage>(ErrorMessage.noError);
```
Comment (growingananas) [inline src/App.tsx:18]: naming  with 'is' prepix is common for boolean data types.

**Review by silchyn — CHANGES_REQUESTED** [review]
What's the reason to display temp todo in the bottom if you put new todo in the top?

File: src/App.tsx (hunk 2 above)
Comment (Shevtsooov) [reply src/App.tsx]: @growingananas Hey, thanks for the good point)
The variable has been renamed.

**Comment by Shevtsooov** [comment]
@silchyn Hi, you are right. Corrected!

**Review by silchyn — APPROVED** [review]
The next part of this task will be reviewed

**Pull request closed without merging**